Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -450,6 +460,17 @@ static String mutationObserverNotificationString()
         m_overflowControlsContainer->setAnchorPoint({ });
         m_rootLayer->addChild(*m_overflowControlsContainer);
     }
+
+#if ENABLE(UNIFIED_PDF_SELECTION_LAYER)
+    if (!m_selectionLayer) {
+        m_selectionLayer = createGraphicsLayer("PDF selections"_s, isFullMainFramePlugin() ? GraphicsLayer::Type::PageTiledBacking : GraphicsLayer::Type::TiledBacking);
```

Comment:
Just unconditionally use a `GraphicsLayer::Type::TiledBacking`.

Reply:
Addressed in https://github.com/WebKit/WebKit/commit/6de504e416889f603b9470e7a2d3f7bda937db05